crates/worker/src/partition/state_machine/effects.rs

```diff
@@ -416,24 +416,27 @@ impl Effect {
                 span_context,
                 ..
             } => match timer_value.value() {
-                Timer::CompleteSleepEntry(_) => {
+                Timer::CompleteSleepEntry(invocation_id, entry_index) => {
                     info_span_if_leader!(
                         is_leader,
                         span_context.is_sampled(),
                         span_context.as_parent(),
                         "sleep",
-                        restate.invocation.id = %timer_value.invocation_id(),
-                        restate.timer.key = %TimerKeyDisplay(timer_value.key()),
+                        restate.invocation.id = %invocation_id,
```

Comment:
This is unnecessary repetition I think, you have the invocation id in the span already